ts/features/payments/checkout/hooks/usePagoPaPayment.ts

```diff
@@ -52,9 +50,6 @@ const usePagoPaPayment = (): UsePagoPaPayment => {
   const navigation = useIONavigation();
 
   // Checks if the new wallet section is enabled
```

Comment:
ditto

Reply:
Sorry I don't understand 🤔🤔

Reply:
did you mean ‘as above’? Anyway, I deleted the comment!